environments/development.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   analytics: 'UA-111887945-1',
   apiUrl2: 'https://appimagegenerator-pre.azurewebsites.net',
   androidPackageGeneratorUrl: 'https://pwabuilder-cloudapk-pre.azurewebsites.net',
-  testAPIUrl: 'https://pwabuilder-tests.azurewebsites.net/api'
+  testAPIUrl: 'https://pwabuilder-tests-dev.azurewebsites.net/api'
```

Comment:
Is this the URL we want in prod?

Reply:
Thats fot the develpment environments file, so its not used in prod (: